src/spec.rs

```diff
@@ -1,4 +1,5 @@
 use crate::result::{ZipError, ZipResult};
+use crate::write::{FileOptions, ZipWriter};
```

Comment:
Imports that are only used in doc comments should be either guarded by `#[cfg(doc)]` or, as I favor, replaced with fully-qualified names at the point of use.

Reply:
Done!